tools/benchmark/src/mocha/runnerUtilities.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { Test } from "mocha";
+
+import type { BenchmarkResult, BenchmarkError, BenchmarkData, CustomData } from "../ResultTypes";
+import { timer } from "../timer";
+
+/**
+ * Wraps a function that returns CustomData, measuring its execution time
+ * and capturing either its result or exception.
+ * Returns a callback suitable for passing to emitResultsMocha.
+ * This is a generic utility that is neither mocha-specific nor time benchmark-specific.
+ */
+export function captureResults(
```

Comment:
this function does not depend on mocha, and is useful for non-mocha tests, so it should be outside the "mocha" folder. Maybe in tools/benchmark/src/RunnerUtilities.ts?